xls/build_rules/xls_ir_rules.bzl

```diff
@@ -197,6 +197,7 @@ def _optimize_ir(ctx, src):
     opt_ir_tool = get_executable_from(get_xls_toolchain_info(ctx).opt_ir_tool)
     opt_ir_args = dict(ctx.attr.opt_ir_args)
     IR_OPT_FLAGS = (
+        "top",
```

Comment:
can we refactor this to use separate rules or the `dslx_top` argument, to decouple this work from https://github.com/google/xls/pull/964?